src/react-native/04-intro-to-jsx/intro-to-jsx.md

```diff
@@ -235,7 +235,7 @@ If you’ve implemented the solution correctly, the tests will pass when you run
 
 Next, we want to render a list of state names in our application:
 
-<img alt="Screenshot of a mobile application interface text “Place My Order: Coming Soon To...”, “Illinois“, and “Wisconsin“." src="../static/img/react-native/04-intro-to-jsx/02-solution.png" style="max-height: 750px; border: 4px solid black; border-radius: 25px;"/>
+<img alt="Screenshot of a mobile application interface text “Place My Order: Coming Soon!”, “Illinois“, and “Wisconsin“." src="../static/img/react-native/04-intro-to-jsx/02-solution.png" style="max-height: 750px; border: 4px solid black; border-radius: 25px;"/>
```

Comment:
Should we update the image to have the same text too?

Reply:
It’s updated!